packages/components/shared/src/controllers/anchor.ts

```diff
@@ -111,10 +111,10 @@ export class AnchorController implements ReactiveController {
 
     if (anchorElement && !this.#host.hasAttribute('aria-details')) {
       anchorElement.id ||= `sl-anchor-${nextUniqueId++}`;
+      this.#host.id ||= `sl-popover-${nextUniqueId++}`;
```

Comment:
It doesn't always have to be a `<sl-popover>` right? Perhaps change to `sl-anchor-host-${nextUniqueId++}`?

Reply:
But this is the replacement for the popovertarget functionality according to the comment above the function. And it isn't referring to the sl-popover element, but it's a prefixed id. I think it's more confusing to name it something else.